README.md

```diff
@@ -223,6 +223,13 @@ Run a Slack slash command. Simply prepend `/slack slash` to what you'd type in t
 /slack slash /desiredcommand arg1 arg2 arg3
 ```
 
+Paste in the input the slack link to some message:
+```
+/slack linkarchive [$]thread_id
```

Comment:
Should probably write `message_id` instead, so it doesn't seem like this can only be used for thread messages.

Reply:
Fixed, I also reordered alphabetically the `/key` lines